src/main/java/gift/web/dto/response/ErrorResponse.java

```diff
@@ -0,0 +1,43 @@
+package gift.web.dto.response;
+
+import java.time.LocalDateTime;
+import org.springframework.validation.BindingResult;
+
+public class ErrorResponse {
+
+    private String code;
+    private String field;
+    private String description;
+    private LocalDateTime timestamp;
```

Comment:
시간에 대해서 기록한 이유가 어떻게 될까요?

Reply:
로깅할 때 구체적인 시간이 기록되어있어 로그를 찾기 수월하다고 생각합니다. 또 다른 장점이 있나요?

https://softwareengineering.stackexchange.com/questions/420911/what-are-the-advantages-of-sending-timestamp-in-the-response

다음과 같은 글을 찾아봤는데 답변으로 "It can help with identifying latency issues" 라고 나와있었습니다.
지연 문제의 경우 개발자도구에서 확인할 수 있는데 이 부분에 대해서는 잘 모르겠습니다 혹시 알려주실 수 있나요? 
좀 더 보기 편하게 알 수 있다는 점에서 이점이 있다고 이야기 한 것일까요?

<img width="1026" alt="image" src="https://github.com/kakao-tech-campus-2nd-step2/spring-gift-wishlist/assets/89574219/4ccb71f8-c956-4a0b-9fe1-fb785ebf882d">

Reply:
글에서 "It can help with identifying latency issues"말은 응답값에 타임스탬프 (시간) 이 있으니 말은 요청 -> 응답까지 오는 시간에 대한 지연이 생길 때 파악하기 수월하다라는 말인 것 같아요

Reply:
응답값에 타임스탬프가 없어도 로깅하면 로그에 시간이 남지 않나요?